src/gui/sdlmain.cpp

```diff
@@ -3592,7 +3709,11 @@ static void HandleVideoResize(int width, int height)
 
 #if C_OPENGL
 	if (sdl.desktop.window.resizable && sdl.desktop.type == SCREEN_OPENGL) {
-		sdl.clip = calc_viewport(width, height);
+		int canvas_width  = 0;
+		int canvas_height = 0;
```

Comment:
For consistency, suggest: `SDL_Rect canvas = ();`

Reply:
Done.